package/src/components/StripeForm/v1/StripeForm.md

```diff
@@ -0,0 +1,18 @@
+### Overview
+
+#### Usage
+This form is intended to be used to integrate [react-stripe-elements](https://github.com/stripe/react-stripe-elements) into a store front project. The hosting project will need to wrap the the main app component with the `<StripeProvider>` and provide an api key.
```

Comment:
You have typo "the the" in here. Also, can you link to the Stripe instructions, right to the section about how to add the `<StripeProvider>`, and maybe also call out the section about how to do it with SSR.